setup.h.in

```diff
@@ -349,7 +349,8 @@
 #endif
 
 
-#ifdef _MSC_VER
+#if defined(_MSC_VER) || \
+    (defined(__MINGW32__) && (__GNUC__ > 4 || __GNUC_MINOR__ >= 8))
 #define wxUSE_GRAPHICS_CONTEXT 0
```

Comment:
This same test has been used in all the above versions of setup0.h to determine when to set wxUSE_GRAPHICS_CONTEXT to 1.  So why is it setting wxUSE_GRAPHICS_CONTEXT to 0 here under the same conditions?  Perhaps I'm not understanding the purpose of setup.h.in; maybe we are just allocating storage locations here for all variables; I see that just about all variables are initialized to zero.  But if that's the case, then why bother applying any conditionals here at all?

Reply:
`setup.h.in` is automatically generated (see `build/update-setup-h`), so it keeps the conditions present in the other files, even if they're not needed here.

Reply:
I just did a pull to get these updates, and was reading up on the GNU C preprocessor constants:

https://gcc.gnu.org/onlinedocs/cpp/Common-Predefined-Macros.html

While I doubt this will be an issue (no one is likely still using 2.95, and apparently version 3 never went above 3.4.6), looks like the suggested way to write the GNUC condition would be:

```
#if __GNUC__ > 4 || \
    (__GNUC__ == 4 && __GNUC_MINOR__ >= 8)
```
The GNU C releases are identified here: 

https://www.gnu.org/software/gcc/releases.html

Reply:
Yes, I was lazy here, but current wxWidgets sources definitely won't compile with gcc < 3 and gcc 3.8 didn't exist, so this isn't really a problem.